cpp/cluster/ClusterProperties.cc

```diff
@@ -18,10 +18,11 @@ namespace freud { namespace cluster {
     compute loops over all points in the given array and determines the center
     of mass of the cluster as well as the gyration tensor. These can be
     accessed after the call to compute with getClusterCenters() and
-    getClusterGyrations().
+    getClusterInertiaMoments().
```

Comment:
`getClusterGyrations` is what gets the gyration tensors, right?

Reply:
Yes.